frappe/tests/test_oauth20.py

```diff
@@ -367,6 +367,7 @@ def decode_id_token(self, id_token):
 			audience=self.client_id,
 			key=self.client_secret,
 			algorithms=["HS256"],
+			options={"verify_signature": True, "require": ["exp", "iat", "aud"]},
```

Comment:
```suggestion
			options={"require": ["exp", "iat", "aud"], "verify_exp": True, "verify_iat": True, "verify_aud": True},
```
this seems like a much stronger test? though the current thing is also fine imo

PS: `verify_signature` is `True` by default

Reply:
seems like its just checking for types - can be ignored :P

Thanks for this.